NVDSuppressions.xml

```diff
@@ -9,4 +9,121 @@
+   file name: GetDocument.Insider.dll
+  CVE References ASP.Net versions 1.0-1.1. ESS is built on ASP.Net Core 3.1+, not classic ASP.Net.
+  See https://nvd.nist.gov/vuln/detail/CVE-2005-2224 and https://www.securityfocus.com/bid/14217 for more details.
+   ]]>
+    </notes>
+    <cve>CVE-2005-2224</cve>
+  </suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+   file name: GetDocument.Insider.dll
+   Vunerability applies to AfterLogic MailBee WebMail Pro 4.1 for ASP.NET which is not used by the ESS
+   ]]>
+    </notes>
+    <cve>CVE-2008-0333</cve>
+  </suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+   file name: GetDocument.Insider.dll
+   Suppressing this CVE as it is a .NetCore runtime issue affecting .NetCore runtimes below .NET Core 3.1.4 or lower. The ESS Runs in an AppService which will have the fixes already applied, and the global.json has been addeded to require .NetCore 3.1.408 or greater.
+   More details can be found here: https://github.com/dotnet/announcements/issues/156
+   ]]>
+    </notes>
+    <cve>CVE-2020-1108</cve>
+  </suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+    file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    See https://github.com/dotnet/announcements/issues/175 for full MS Details on CVE.
+    .NET Core and Visual Studio Denial of Service Vulnerability. 
+    This CVE applies to any .NET Core 3.1 but patch is provided at 3.1.405, ESS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
+    </notes>
+    <cve>CVE-2021-1721</cve>
+  </suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+    file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    See https://github.com/dotnet/announcements/issues/194 for full MS Details on CVE.
+    A denial of service vulnerability exists in .NET 5.0, .NET Core 3.1 and .NET Core 2.1 where .NET (Core) server applications providing WebSocket endpoints could be tricked into endlessly looping while trying to read a single WebSocket frame.
+    This CVE applies to any .NET Core 3.1 but patch is provided at 3.1.412, ESS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
+    </notes>
+    <cve>CVE-2021-26423</cve>
+  </suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+    file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    See https://github.com/dotnet/announcements/issues/178 for full MS Details on CVE.
+    A remote code execution vulnerability exists in .NET 5 and .NET Core due to how text encoding is performed.This vulnerability is for System.Text.Encodings.Web and fixed on 5.0.1 and we are using 5.0.1
+    This CVE applies to any .NET Core 3.1 but patch is provided at 3.1.407, ESS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
+    ]]>
```

Comment:
Please update suppression note as 'ENS is pinned to .Net core SDK > 6.0.101'

Reply:
updated accordingly.